.circleci/config.yml

```diff
@@ -210,6 +238,12 @@ workflows:
           env_name: production
           requires:
             - tag_production
+      - deploy_image_to_live:
+          name: deploy-production-live
+          env_name: production
+          requires:
+            - tag_production
+
```

Comment:
Also no need for this workflow job just yet, as we don't have a production namespace on `live`.